browser/src/control/jsdialog/Widget.SidebarContainers.ts

```diff
@@ -79,6 +79,7 @@ JSDialog.panel = function (
 				aria: { label: moreOptionsText },
 				icon: app.LOUtil.getIconNameOfCommand('morebutton'),
 				text: moreOptionsText,
+				noLabel: true,
```

Comment:
good to have that explicitly set, but root cause commit for regression added above "text" field, is that needed?
isn't aria.label enough?

Reply:
ok, fine